doc/introduction/interfaces.rst

```diff
@@ -401,25 +401,25 @@ At the moment, it takes into account the following parameters:
 +==================+==============================+==============+===============+==============+==============+===============+================+================+=============+=============+=============+
 | **Interface**    |**Differentiation method**    | state        |density matrix |  probs       | sample       |expval (obs)   | expval (herm)  | expval (proj)  | var         | vn entropy  | mutual info |
 +------------------+------------------------------+--------------+---------------+--------------+--------------+---------------+----------------+----------------+-------------+-------------+-------------+
-| ``None``         | ``"device"``                 |    :rd:`1`   |      :rd:`1`  |   :rd:`1`    | :rd:`1`      |  :rd:`1`      | :rd:`1`        | :rd:`1`        | :rd:`1`     | :rd:`1`     | :rd:`1`     |
+| ``None``         | ``"device"``                 |    :rd:`1`   |      :rd:`1`  |   :rd:`1`    | :rd:`9`      |  :rd:`1`      | :rd:`1`        | :rd:`1`        | :rd:`1`     | :rd:`1`     | :rd:`1`     |
```

Comment:
This table has effectively two changes:
1. The column for `sample` (all red) is changed from a number of reasons to the reason `9`, which is a fundamental reason why we do not support differentiation of `sample`. All the other reasons are more implementation/software specific, so I would consider them less fundamental reasons why this column is not supported (red).
2. The field `"tf"` + `"adjoint"` + `probs` is marked with 6 (red), but it actually is supported and works, just like for torch or Jax.